CentralComputing/Pod.h

```diff
@@ -0,0 +1 @@
+int pod(int argc, char** argv);
```

Comment:
Should this become a class? 
Either make this a class, with the current pod() becoming the constructor (make the default constructor private)
Or, make this static.